### PR TITLE
Change Include pattern to include all javascript files

### PR DESCRIPTION
**Pull request by geoperez**

### CODE REVIEW

File: files/tubular/update.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
   "repo": "unosquare/tubular",
   "files": {
     "basePath": "dist/",
-    "include": ["tubular-bundle.min.js","tubular-bundle.min.css"]
+    "include": ["*.js","tubular-bundle.min.css"]
```
Comment (jimaek) [inline files/tubular/update.json]: how about using `*.min.js` instead? Its better for production

Comment (geoperez) [inline files/tubular/update.json]: Sure, much better

**Comment by geoperez** [comment]
Do I need to pull a new request?

**Comment by jimaek** [comment]
No, you can update this one

**Comment by geoperez** [comment]
Done

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**